### PR TITLE
Update braintree-vault.md

### PR DESCRIPTION
**Pull request by RakeshJesadiya**
Request data is invalid. Syntax Error: Expected Name, found <EOF>".

## Purpose of this pull request

This pull request (PR) contains the updated setPaymentMethodOnCart() mutation. Resolved syntax error from the Request. Given link to the braintree_cc_vault object.

## Affected DevDocs pages

https://devdocs.magento.com/guides/v2.3/graphql/payment-methods/braintree-vault.html
https://devdocs.magento.com/guides/v2.4/graphql/payment-methods/braintree-vault.html

### CODE REVIEW

File: src/guides/v2.3/graphql/payment-methods/braintree-vault.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ The following diagram shows the workflow for placing an order when Braintree Vau
 
 1. The client renders the token information, and the customer selects a payment method.
 
-   When the customer selects a stored payment method, the PWA uses the [`setPaymentMethodOnCart`]({{page.baseurl}}/graphql/mutations/set-payment-method.html) mutation to set the payment method to `braintree_cc_vault`. The vaulted public hash is passed with other optional properties in the [`braintree_cc_vault`](#braintree_cc_vault-object).
+   When the customer selects a stored payment method, the PWA uses the [`setPaymentMethodOnCart`]({{page.baseurl}}/graphql/mutations/set-payment-method.html) mutation to set the payment method to [`braintree_cc_vault`](#braintree_cc_vault-object). The vaulted public hash is passed with other optional properties in the [`braintree_cc_vault`](#braintree_cc_vault-object).
```
Comment (BarnyShergold) [inline src/guides/v2.3/graphql/payment-methods/braintree-vault.md:23]: Is this second link really necessary as it also appears elsewhere in the sentence? 

I'd say move the link from the second to the first instance.

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
Just a small suggestion

File: src/guides/v2.3/graphql/payment-methods/braintree-vault.md (hunk 1 above)
Comment (RakeshJesadiya) [reply src/guides/v2.3/graphql/payment-methods/braintree-vault.md:23]: It's not necessary but we can keep it for smooth user experience.

**Comment by RakeshJesadiya** [comment]
@BarnyShergold I have made updated the changes. Thank you.

**Review by keharper — APPROVED** [review]

**Pull request merged**